src/RealtimeClient.ts

```diff
@@ -66,6 +75,7 @@ export default class RealtimeClient {
   apiKey: string | null = null
   channels: RealtimeChannel[] = []
   endPoint: string = ''
+  httpEndpoint: string = ''
```

Comment:
Also this parameter too. Is there another PR that makes use of them?

Reply:
this should be used for http broadcast (aka non ws connection broadcast for edge functions). It will also be useful for longpoll fallback